### DISCUSSION TITLE
rAudio 20221007: System page could not display

### DISCUSSION THREAD

**Discussion by GDV0** [rern/rAudio#1055] [Fixed Issues]
Hello rern,
I remain impressed with the huge work done on this project.
Today, I have updated my functional rAudio (rev 20220917) with the last revision 20221007.
If I click on System in the settings menu, unfortunately System page is not displayed but replaced by this: 
![image](https://user-images.githubusercontent.com/1609438/194720861-30c2dac3-b9e7-47ff-93a8-7560cd6f0894.png)

Something in the previous system settings file seems wrong to the new rev.

**Comment by rern** [comment]
The value of APM, Advanced Power Management, of your HDD is "not supported" which must be replaced as false.
Try:
```sh
wget https://github.com/rern/rAudio-1/raw/UPDATE/srv/http/bash/settings/system-data.sh -O /srv/http/bash/settings/system-data.sh
```

**Comment by GDV0** [comment]
Pefectly fixed !

Thanks a lot
